3737/2/helpers/user_helper.rb

```diff
@@ -0,0 +1,10 @@
+require 'yaml'
+require_relative '../model/user_class'
+
+# class that check user
+class UserHelper
+  def self.number_exists?(enter_number)
+    @number_exists ||= YAML.load_file('data/stud.yml').fetch('numbers', [])
```

Comment:
bad naming.  variable name should be a noun. I expect something like existed_students_numbers